src/app/home/home.component.html

```diff
@@ -21,12 +21,16 @@
 </mat-menu>
 <!--Notification dropdown menu-->
 <mat-menu #notificationMenu="matMenu" [overlapTrigger]="false">
-  <span class="menu-item-notification" mat-menu-item *ngFor="let notification of notifications" (click)="readNotification(notification.doc)">
+  <a routerLink="{{notification.doc.link}}"    

```

Comment:
Are there instances where we would not have this link go to the 'notification/view' route?  If the menu links always lead to the 'notification/view', we should just have that here.